demisto/client.py

```diff
@@ -83,7 +86,19 @@ def CreateIncident(self, inc_name, inc_type, inc_severity, inc_owner, inc_labels
 
     def SearchIncidents(self, page, size, query):
         data = {'filter': {'page': page, 'size': size, 'query': query, 'sort': [{'field':'id', 'asc': False}]}}
-        r = self.req("POST", "incidents/search", data)
-        if r.status_code != 200:
+        response = self.req("POST", "incidents/search", data)
+        if response.status_code != 200:
```

Comment:
If you're changing, might as well change to built-in constant instead of 200

Reply:
done